src/libraries/Microsoft.PowerFx.Core/Public/Types/AggregateType.cs

```diff
@@ -122,11 +122,18 @@ public IEnumerable<NamedFormulaType> GetFieldTypes()
         {
             return FieldNames.Select(field =>
             {
-                var fieldType = GetFieldType(field);
-                
                 var displayName = DType.TryGetDisplayNameForColumn(_type, field, out var dName)
                     ? dName
                     : null;
+
+                if (this.TryGetBackingDType(field, out var backingDType))
+                {
```

Comment:
@jas-valgotar  - this one is worth double checking - it mirrors change you made for RecordValues.